### ISSUE TITLE
Builds have been failing after adding a file in composer.json

### ISSUE THREAD

**Issue by andriiosmak** [Shippable/support#5066]
### Description of your issue:
My builds have been failing since the moment  I added the following to my composer.json.
```
    "autoload": {
        "files": [
            "app/helpers.php"
        ],
```
It looks like it can't find the find while running `composer install` command.
`Fatal error: composerRequire5c746643f5a1654f83bdb68677f575de(): Failed opening required '/root/src/github.com/andriiosmak/cloudlog/vendor/composer/../asm89/stack-cors/helpers.php' (include_path='.:') in /root/src/github.com/andriiosmak/cloudlog/vendor/composer/autoload_real.php on line 66`

It tries to find the file in `/root/src/github.com/andriiosmak/cloudlog/vendor/composer/../asm89/stack-cors/helpers.php` although it should search it in `/root/src/github.com/andriiosmak/cloudlog/app/`

- Link to the build failure
https://app.shippable.com/github/andriiosmak/cloudlog/runs/62/1/console
- Scripts called from the yml (optional)
```
language: php

php:
  - 7.3

before_script:
  - cp .env.testing .env
  - composer install

script:
  - vendor/bin/phpcs --extensions=php --standard=PSR12 app --ignore=*.blade.php
  - vendor/bin/phpmd app text codesize,unusedcode,naming
  - vendor/bin/phpcpd app
  - vendor/bin/phpunit --no-coverage

```

**Comment by a-murphy** [comment]
Does this composer.json file work locally?  Or have any other files specified by path that work?  I assume `helpers.php` may be a fairly common file name, so I can't be sure that the error is even about the same file.  Could it be that there is another dependency missing?

**Comment by andriiosmak** [comment]
Yep, locally it works as expected. I started getting the exception once I added that file, so it is 100% related to it.

**Comment by andriiosmak** [comment]
It looking for `helpers.php` in`vendor/asm89/stack-cors/` instead of `app/` as defined in `composer.json`.

**Comment by a-murphy** [comment]
Have you tried updating Composer with `composer self-update`?

If that doesn't work, does it work in a new local clone of your project?  It's not uncommon for issues to be opened because a file wasn't committed.  There's not really much I can do because I don't know anything about your `app/helpers.php` or how it might be related to `asm/stack-cors`.  It could be that `asm/stack-cors` just happened to be the first dependency, or it could be that the two are related.

**Comment by andriiosmak** [comment]
Yes, it is the first dependency.

**Comment by andriiosmak** [comment]
Fixed the issue by renaming `helpers.php`. Not sure what was the cause of the issue though.
